packages/core/src/App/Components/Elements/Modals/mul-cfd-upgrade-modal.jsx

```diff
@@ -259,6 +289,8 @@ const UpgradeAccountsModal = ({ is_upgrade_modal_visible, toggleUpgradeAccounts,
                         <ModalContent
                             openRealAccountSignup={openRealAccountSignup}
                             toggleUpgradeAccounts={toggleUpgradeAccounts}
+                            show_multiplier={true}
```

Comment:
this flags should come from outside of the modal to disable
Like `<UpgradeAccountsModal show_multiplier={false}/>`
and if we don't want to disable one of the columns it should use the default value of props 
Like 
```
const UpgradeAccountsModal = ({ is_upgrade_modal_visible, toggleUpgradeAccounts, openRealAccountSignup, show_multiplier=true, show_cfd=true  }) => {
```
And then use the props like this
```
                     <ModalContent
                            openRealAccountSignup={openRealAccountSignup}
                            toggleUpgradeAccounts={toggleUpgradeAccounts}
                            show_multiplier={show_multiplier}
```